src/fragments/lib/graphqlapi/js/setup/connect.mdx

```diff
@@ -644,45 +607,25 @@ const promise = makeAPICall();
-> - Your application connected to the API
-> - Populated data to query
+You can read application data using the Amplify GraphQL client. In this next section we will review the difference between listing data and getting data, how to filter query results to get just the data you need, and how to paginate results to make you're data more manageable. We will also show you how to cancel these requests when needed. Before you begin you will need your application connected to the API and some populated data to query.
 
 ### Read and filter your data
-
-[comment]: # (Short explanation of the difference between listing and getting data.)
   
-Queries are used to read data from the API and include the list and get operations.
+Queries are used to read data from the API and include the `List` and `Get` operations.
   
 You can use both `List` and `Get` to query data in GraphQL. A `List` will get you a record of several items, such as Todo items, and does not require an ID. This is best suited for getting an overview or summary of items or can help when filtering by specific criteria across a list of items. When you want to query a single entry by ID, you would use `Get` to retrieve a specific Todo item.
-  
-#### Learn more about limitations and costs for specific operations
-  
-[comment]: # (EXPANDER SECTION that provides an overview of the cost of specific operations - e.g., list vs. get, queries, and scans; as well as limitations for specific functions - e.g., sorting, open search, items, and filters. Could not find this – part of this? https://aws.amazon.com/appsync/pricing/.)
-  
-[comment]: # (EXPANDER end)
-[comment]: # (RENE, any content for the limitations and costs section?)
 
-[comment]: # (Steps to list or get data. This is then followed by an explanation of how to filter data by variable and steps to filter data. Taken from 1- https://github.com/aws-amplify/docs/blob/main/src/fragments/lib/graphqlapi/js/query-data.mdx and 2- https://github.com/aws-amplify/docs/blob/main/src/pages/cli/graphql/overview.mdx)
+<Callout>
+
+__Note:__ The cost structure of your underlying data source can impact the cost to run some queries. For example, the `List` operation uses "scan operations" which uses more read request units than the `Get` operation. You will want to review the associated costs for these operations for your data source. In our example case here we are using Amazon DynamoDB and you can [learn more about how DynamoDB pricing is calculated](https://aws.amazon.com/dynamodb/pricing/) on their site.
+
+</Callout>
 
-The Amplify CLI's codegen automatically creates code for all possible GraphQL operations—mutations, queries, and subscriptions. For JavaScript applications, this generated code is saved in the src/graphql folder.
+The Amplify CLI's codegen automatically creates code for all possible GraphQL operations—mutations, queries, and subscriptions. For JavaScript applications, this generated code is saved in the src/graphql folder. 
```

Comment:
Again general question about Codegen is it only available for JS/TS - should we be considering this section is replaced for other frameworks?

Reply:
it's available for all frameworks but I think we need to revisit where to "place" this content in general.